plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -222,6 +224,13 @@ bool AsyncPluginImpl::NewWidget(ScanlimeFadecandy *widget) {
           "fadecandy-" + widget->SerialNumber()));
 }
 
+bool AsyncPluginImpl::NewWidget(ShowJockeyDMXU1 *widget) {
+  return StartAndRegisterDevice(
+      widget,
+      new GenericDevice(m_plugin, widget, "ShowJockey-DMX-U1 Device",
```

Comment:
Can you display the serial here like e.g. https://github.com/nbeAnomes/ola/blob/83ccf21f76fd7e15e0a1b2bb454f35ffd9a3aa0d/plugins/usbdmx/AsyncPluginImpl.cpp#L223